framework/include/mesh/FaceInfo.h

```diff
@@ -208,14 +211,20 @@ FaceInfo::neighbor() const
 }
 
 inline FaceInfo::VarFaceNeighbors
-FaceInfo::faceType(const std::string & var_name) const
+FaceInfo::faceType(const std::pair<unsigned int, unsigned int> & var_sys) const
 {
-  auto it = _face_types_by_var.find(var_name);
+  auto it = _face_types_by_var.find(var_sys);
   if (it == _face_types_by_var.end())
-    mooseError("Variable ", var_name, " not found in variable to VarFaceNeighbors map");
+    mooseError("Variable ", var_sys.first, " not found in variable to VarFaceNeighbors map");
```

Comment:
need more context than the system number in that output message

Reply:
I don't know if we do. A user should never hit this. I'd be tempted to just change this to an assertion

Reply:
Either an assertion or add the system number to the error message. I don't have a means to get back the variable name since I don't have a reference or pointer to the problem or system

Reply:
system number will have to do